server/db/load-postgres.js

```diff
@@ -0,0 +1,75 @@
+const CONFIG = require('../../config.js');
+const pg = require('pg');
+const path = require('path');
+
+const client = new pg.Client({
+  user: CONFIG.POSTGRES.USER,
+  host: CONFIG.POSTGRES.URL,
+  database: CONFIG.POSTGRES.DBNAME,
+  password: CONFIG.POSTGRES.PASS,
+  port: CONFIG.POSTGRES.PORT,
+});
+
```

Comment:
Code dryness could be improved, but honestly this is file that should only be run once; it doesn't rise to the level of concern.